### DISCUSSION TITLE
License Audit does not consider all projects in the mono repository

### DISCUSSION THREAD

**Discussion by shyim** [JetBrains/Qodana#60] [Q&A] [answered]
I have a project structure

```
> src/Administration/package.json
> src/Storefront/package.json
> composer.json
```

Qodana looks only into the composer.json in the root

**Comment by tiulpin** [comment, marked as answer]
Try using the following option: https://www.jetbrains.com/help/qodana/qodana-yaml.html#Monorepo+support

**Reply by shyim** [reply]
It did worked thanks

**Reply by shyim** [reply]
@tiulpin the doc seems to be removed. Is this still supported 🤔

**Reply by tiulpin** [reply]
Hi, we finally integrated the feature to all our main linters – https://blog.jetbrains.com/qodana/2022/05/keep-your-dependency-licenses-in-check

Since then, unfortunately, you have to set up three separated builds for your monorepo. Thank you!

**Reply by shyim** [reply]
So I can expect that this functionality is added sometime again back? 😅

**Reply by tiulpin** [reply]
We will do something on integrating JS to other linters (so it could be installed as a plugin), but for now it's not on the roadmap
